styles/styles.css

```diff
@@ -484,6 +484,7 @@ main .section.shadow  {
   background: var(--background-color-light-orange);
   box-shadow: 0 1.25rem 2.5rem var(--box-shadow);
   padding: 1.8rem 0;
+  margin: 2.5rem 0;
```

Comment:
@ravverma have you checked any other block is not having any side effect because of this ?

Reply:
yes , It was found when I have aded block in equity doc.
and this variant class is created for `trending-news`